charter/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Font and Text Community Group Charter
+---
+This charter:  {{ page.url | remove: "index.html" | prepend: site.url }}
+
+Feedback about this document:
+    https://github.com/w3c/font-text-cg/issues
+
+## Goals
+
+The Font and Text community group gathers individuals and organizations
+interested in discussing and developing specifications and implementations
+for technologies
+which operate on and at the interface between
+text encoding and font formats.
+Examples of such technologies are
+shaping engines and text layout applications.
+
+## Scope of work
+
+Under the present charter, the Community Group will
+determine a structure and process
+under which such specifications may be created
```

Comment:
I agree this should be clarified, it is no longer as clear as it was in early drafts what this connects to. Do you have exact wording in mind?

Reply:
Is it really up to a CG to determine a process by which the work is done? Do we have to follow the W3C Process (at least parts of it relevant to CGs) when the work is done under W3C umbrella? I think something similar would be the case for other venues as well.

Reply:
Please see the [background message linked above sent to the MPEG-SPEC list back in August](https://lists.aau.at/pipermail/mpeg-otspec/2020-August/002163.html), especially the clarification about item 4. The entire point of this plan is to form a CG with a charter that specifically does *not* have any technical specifications as deliverables so industry wide players can freely use this as a discussion hub and work from there to figure out what needs to be spun for for specific deliverables. This has been done in consultation with people at the W3C, and yes it's up to this CG's charter to map that out, but yes of course we can only work inside the scope of broader W3C policy for CGs. We still have to define our scope to *be* a CG.

Reply:
My comment is specifically related to the part that references "such specifications" (which is something defined by "Goals" section so I agree with @lianghai that clarification is needed). Since we expect that this CG will determine what specifications will need to be developed and what is the right venue for them, it is the choice of venue that will to significant extent define the future process  by which the work will be done. I think we need to clarify this as well.